Libraries/HyperV.psm1

```diff
@@ -37,7 +37,12 @@ function Create-AllHyperVGroupDeployments($SetupTypeData, $GlobalConfig, $TestLo
         $HyperVHostArray = @()
         if ($deployOnDifferentHosts -eq "yes") {
             foreach ($HypervHost in $GlobalConfig.Global.HyperV.Hosts.ChildNodes) {
-                $HyperVHostArray += $HyperVHost.ServerName
+                if (!($HyperVHostArray -contains $HyperVHost.ServerName)) {
```

Comment:
Help me understand what this code change means? I see Hyper-V OOB document said it allows multiple hosts like ' -TestLocation “server_1,server_2”' Any example falling into this false block?

Reply:
I see. Then out GlobalConfiguration xml file should be updated with your information. The current data is both 'localhost'. It should be 'localhost1' and 'localhost2'. Am I understanding correct?

Reply:
I am not sure I understand your last comment.